src/modules/commander/commander_params.c

```diff
@@ -968,3 +968,17 @@ PARAM_DEFINE_FLOAT(COM_LKDOWN_TKO, 3.0f);
 * @value 1 Enabled
 */
 PARAM_DEFINE_INT32(COM_ARM_ARSP_EN, 1);
+
+/**
+ * Enable FMU SD card detection check
+ *
+ * This check detects if the FMU SD card is missing.
+ * Depending on the value of the parameter, the check can be
+ * disabled, warn only or deny arming.
+ *
+ * @group Commander
+ * @value 0 Disabled
+ * @value 1 Warning only
+ * @value 2 Enforce SD card presence
+ */
+PARAM_DEFINE_INT32(COM_ARM_SDCARD, 1);
```

Comment:
What about boards w/o SD card?

Reply:
is this detail available through the board config or some board specific defines? if not, how would you suggest to address it? keep the check disabled by default?

Reply:
There's `CONFIG_MMCSD` on NuttX boards, but there are board variants w/o sd card that have it defined.
Warning by default is fine with me though.